processor/schemaprocessor/internal/translation/translation_race_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package translation
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/schemaprocessor/internal/fixture"
+)
+
+func TestRaceTranslationSpanChanges(t *testing.T) {
+	t.Parallel()
+
+	tn, err := newTranslatorFromReader(
+		zap.NewNop(),
+		"https://example.com/1.7.0",
+		LoadTranslationVersion(t, "complex_changeset.yml"),
+	)
+	require.NoError(t, err, "Must not error when creating translator")
+
+	fixture.ParallelRaceCompute(t, 10, func() error {
```

Comment:
Where in the current implementation do we have concurrency that needs race testing? I am probably missing it.

Reply:
I don't think there is any to be honest - a lot of this part's code is the contribution of @MovieStoreGuy so maybe he could speak as to his intention.  Within this part, there's no concurrency.  In the full processor, a lot of this code can be called under concurrent conditions (processors in general get called concurrently).  The `Manager` piece gets called concurrently and has some logic around that.